not enough arguments in call to cbor.NewUnmarshallerAtlased

When i go get github.com/ipfs/go-ipld-cbor, the error occurs:
C:\Users\ztwil\go\src\github.com\ipfs\go-ipld-cbor\encoding\unmarshaller.go:29:43: not enough arguments in call to cbor.NewUnmarshallerAtlased
        have (*proxyReader, atlas.Atlas)
        want (cbor.DecodeOptions, io.Reader, atlas.Atlas)

--- a/encoding/unmarshaller.go
+++ b/encoding/unmarshaller.go
@@ -26,7 +26,7 @@ type Unmarshaller struct {
 // NewUnmarshallerAtlased creates a new reusable unmarshaller.
 func NewUnmarshallerAtlased(atl atlas.Atlas) *Unmarshaller {
 	m := new(Unmarshaller)
-	m.unmarshal = cbor.NewUnmarshallerAtlased(&m.reader, atl)
+	m.unmarshal = cbor.NewUnmarshallerAtlased(cbor.DecodeOptions{CoerceUndefToNull: true}, &m.reader, atl)
 	return m
 }
 

--- a/package.json
+++ b/package.json
@@ -39,9 +39,9 @@
     },
     {
       "author": "why",
-      "hash": "QmfWqohMtbivn5NRJvtrLzCW3EU4QmoLvVNtmvo9vbdtVA",
+      "hash": "QmNScbpMAm3r2D25kmfQ43JCbQ8QCtai4V4DNz5ebuXUuZ",
       "name": "refmt",
-      "version": "1.1.2"
+      "version": "1.1.3"
     }
   ],
   "gxVersion": "0.10.0",

